src/system/NewTabs/Tabs.stories.jsx

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
I saw radix supports orientation vertical, do you think it's possible to have or we can do in a future version?

Reply:
@djalmaaraujo do we already have a design example for the vertical tabs? I tried playing around and we can support them but I feel that if we have a design to follow we'll get a better output.

Reply:
@andrea-sdl I don't think so, so feel free to ignore for now